### PR TITLE
CT-310 Update the assign page

### PR DESCRIPTION
**Pull request by aliuk2012**
https://dsdmoj.atlassian.net/browse/CT-310

- Removed green flash notice on the new assignment view
- Added "Assign case" page heading
- Copy change: 'Create assignment' to 'Create and assign case'
- Added a 'Successfully created case' flash message on case list view

### CODE REVIEW

**Comment by aliuk2012** [comment]
@misaka @EdwardAndress  I think we should review and merge this and then tweak the rest of assign page later.

**Comment by aliuk2012** [comment]
I've split this ticket into two separate JIRA tickets and so the understanding work thats blocking this PR/work has been moved into ct-353.

File: app/views/assignments/new.html.slim [hunk 1]
```diff
@@ -1,3 +1,7 @@
+= render partial: 'layouts/header', locals: {page_heading: t('.new_assignment')}
+
 = form_for [ @case, @assignment ] do |f|
-	= f.collection_select :assignee_id, User.drafters, :id, :email
-	= f.submit class: 'button'
+
+  -# f.radio_button_fieldset :assignee_id, choices: User.drafters.tap(&:email).to_sym
```
Comment (misaka) [inline app/views/assignments/new.html.slim]: Do we want this commented-out line?

Comment (aliuk2012) [inline app/views/assignments/new.html.slim]: Fixed it

**Pull request merged**